tests/lib/rules/no-fallthrough.js

```diff
@@ -214,6 +222,59 @@ ruleTester.run("no-fallthrough", rule, {
                     column: 1
                 }
             ]
+        },
+        {
+            code: "switch(foo) { case 0: \n /* with comments */  \ncase 1: b(); }",
+            errors: [
+                {
+                    messageId: "case",
+                    type: "SwitchCase",
+                    line: 3,
+                    column: 1
+                }
+            ]
+        },
+        {
```

Comment:
Can you also add a test case with `allowEmptyCase: false`, and a test case with an empty object (to test default value in the schema), like:

```js
{
    code: "switch(foo) { case 0:\n\ncase 1: b(); }",
    options: [{ allowEmptyCase: false }],
    errors: [
        {
            messageId: "case",
            type: "SwitchCase",
            line: 3,
            column: 1
        }
    ]
},
{
    code: "switch(foo) { case 0:\n\ncase 1: b(); }",
    options: [{}],
    errors: [
        {
            messageId: "case",
            type: "SwitchCase",
            line: 3,
            column: 1
        }
    ]
},
```

Reply:
added